index.bs

```diff
@@ -420,13 +425,22 @@ for the name and value of the header are described by the following ABNF
 grammar [[!RFC5234]]:
 
 <pre dfn-type="grammar" link-type="grammar">
-    suborigin = 1*( <a>LOWERALPHA</a> / <a>DIGIT</a> / "-" )
+    <dfn>suborigin-name</dfn> = 1*( <a>LOWERALPHA</a> / <a>DIGIT</a> / "-" )
+    <dfn>suborigin-policy-option</dfn> = "'unsafe-postmessage-send'"
+                              / "'unsafe-postmessage-receive'"
```

Comment:
Fair point. If everyone that wants to use one wants the other as well, might as well make it one directive. Not sure how to best figure that out though.